winui/Kanban/cards.md

```diff
@@ -0,0 +1,161 @@
+* `Tags` - Used to assign multiple tags to a kanban card, which can be used for categorization, filtering, or additional metadata.
+
+{% highlight C# %}
+
+KanbanModel taskDetail = new KanbanModel();
+taskDetail.Title = "UWP Issue";
+taskDetail.Id = "651";
+taskDetail.Description = "Crosshair label template not visible in UWP";
+taskDetail.Category = "Open";
+taskDetail.IndicatorColorKey = "High";
+taskDetail.Tags = new List<string>() { "Bug Fixing" };
+taskDetail.Image = new Image
+{
+    Source = new BitmapImage(new Uri("ms-appx:///Assets/Kanban/People_Circle1.png"))
+};
+
+{% endhighlight %}
+
+Following code snippet is used to define the colors for each key.
+
+{% tabs %}
+{% highlight XAML hl_lines="6 7 8 9 10" %}
+
+<kanban:SfKanban x:Name="kanban"
+                 ItemsSource="{Binding TaskDetails}">
+    <kanban:SfKanban.DataContext>
+        <local:ViewModel/>
+    </kanban:SfKanban.DataContext>
+    <kanban:SfKanban.IndicatorColorPalette>
+        <kanban:KanbanColorMapping Key="Low" Color="Blue"/>
+        <kanban:KanbanColorMapping Key="Normal" Color="Green"/>
+        <kanban:KanbanColorMapping Key="High" Color="Red"/>
+    </kanban:SfKanban.IndicatorColorPalette>
+</kanban:SfKanban>
+
+{% endhighlight %}
+
+{% highlight C# hl_lines="5" %}
+
+List<KanbanColorMapping> indicatorColorPalette = new List<KanbanColorMapping>();
+indicatorColorPalette.Add(new KanbanColorMapping() { Key="Low", Color = Colors.Blue });
+indicatorColorPalette.Add(new KanbanColorMapping() { Key= "Normal", Color = Colors.Green });
+indicatorColorPalette.Add(new KanbanColorMapping() { Key= "High", Color = Colors.Red });
+this.kanban.IndicatorColorPalette = indicatorColorPalette;
+
+{% endhighlight %}
+{% endtabs %}
+
+## Customizing kanban cards
+
+### Customize kanban cards using template
+
+The appearance of Kanban cards can be fully customized using the `CardTemplate` property of the `SfKanban` control. By setting this property, you can define a custom design for the entire card.
+
+The following code snippet demonstrates how to use the `CardTemplate` property to apply a custom template to Kanban cards:
+
+{% highlight xaml %}
+
+<kanban:SfKanban x:Name="kanban"
+                 ItemsSource="{Binding TaskDetails}">
+    <kanban:SfKanban.CardTemplate>
+        <DataTemplate>
+            <Border BorderBrush="Green" BorderThickness="1" CornerRadius="10" Background="LightGreen">
+                <StackPanel Margin="10">
+                    <TextBlock Text="{Binding Title}" TextAlignment="Center" FontWeight="Bold" FontSize="14" />
+                    <TextBlock Text="{Binding Description}" TextAlignment="Center" FontSize="14" TextWrapping="Wrap" />
+                </StackPanel>
+            </Border>
+        </DataTemplate>
+    </kanban:SfKanban.CardTemplate>
+    <kanban:SfKanban.DataContext>
+        <local:ViewModel/>
+    </kanban:SfKanban.DataContext>
+</kanban:SfKanban>
+
+{% endhighlight %}
+
+### Customize kanban cards using template selector
```

Comment:
Customize card appearance using DataTemplateSelector

You can customize the card appearance by using the `CardTemplateSelector` property in the `SfKanban`. The DataTemplateSelector can choose a DataTemplate at runtime based on the value of a data-bound to kanban card appearance by using the `CardTemplateSelector`. It allows you to choose a different data template for each card, as well as to customize the appearance of a particular card based on certain conditions.